force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -11,5 +11,5 @@ trigger AccountTrigger on Account(
     // For more information on how the framework operates, see the following classes:
```

Comment:
@codefriar maybe we can have a separate example for the one with `MetadataTriggerHandler` than the original one. Let me know what you think? I think that will be good since this is a recipe.

Reply:
> @codefriar This looks really good start!
> 
> I think we should add a proper test case asserting how the order of execution flag works with multiple classes before this is merged.
> 
> Also, I would mention some caveats
> 
> * If org has Managed packages already then this won't help to control their order of execution.
> * This does not allow one to declaratively order the methods of classes. They still need to be managed in the apex class.

The managed package thing is a good caveat to mention, and I've added a comment to that effect.
I'm less concerned about the order of *methods* in the classes so much as the classes themselves. With this pattern in place, it's easy and safe to create small, tactical trigger handler classes for each different division, or story of development. This allows the declarative ordering of those classes, which, if they're adhering to proper standards should only do one thing anyway.